bulkwebhook/bulk_webhook/doctype/kafka_hook/kafka_hook.py

```diff
@@ -79,7 +79,7 @@ def run_kafka_hook(
     if isinstance(doc_list, str):
         doc_list = [doc_list]
 
-    for doc_name in doc_list:
+    for doc_name in set(doc_list):
```

Comment:
@aakvatech I am assuming that order of `doc_list` doesn't matter. Please let me know if that is not the case.